homeassistant/components/sabnzbd/sensor.py

```diff
@@ -23,13 +23,14 @@ class SabnzbdSensor(SensorEntity):
 
     def __init__(self, sensor_type, sabnzbd_api_data, client_name):
         """Initialize the sensor."""
-        self._client_name = client_name
-        self._field_name = SENSOR_TYPES[sensor_type][2]
-        self._name = SENSOR_TYPES[sensor_type][0]
+        meta_data = SENSOR_TYPES[sensor_type]
+        self._field_name = meta_data.key
         self._sabnzbd_api = sabnzbd_api_data
-        self._state = None
+        self._attr_state = None
         self._type = sensor_type
-        self._unit_of_measurement = SENSOR_TYPES[sensor_type][1]
+        self._attr_unit_of_measurement = meta_data.unit_of_measurement
```

Comment:
When setting the entity_description, this line is not longer needed.
```suggestion
```

Reply:
Done